examples/rosettacode/arithmetic_mode.nit

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env nit
+#
+# This file is part of NIT ( http://www.nitlanguage.org ).
+# This program is public domain
+
+# Task: Arithmetic mode
+# SEE: <http://rosettacode.org/wiki/Averages/Mode>
+module arithmetic_mode
+
+# Computes the mode or the `array` values.
+#
+# FIXME: Only the first mode will be returned, should we return an array of modes?
```

Comment:
by returning an array of mod, the nullable can be removed (and replaced by an empty array)